### PR TITLE
4683 - replace tags with newly-designed tags

### PR DESCRIPTION
**Pull request by Miaplacidus**
Currently, our tags use our previous designs. This PR closes out #4683, which asks that we replace our old tags with our newly-designed tags. 

See [here](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=709-8480&mode=design&t=1yOfpMAVIU20mrg3-0) for the new tag designs.

## Link to Issue
Closes: #4683 

## Description of Changes
The tags that have been replaced are: 
1. Element Tags: "Poll", "Snapshot"
2. Stage Tags & Proposal Tags
3. Spam, New, Disabled, & Trending Tags
4. Login tags (previously referred to as "discord tags")
5. Input tags

## Test Plan
- Visit the following pages to see:
  * examples of 1 - 3: [All discussions](https://commonwealth-demo.herokuapp.com/1inch/discussions), [Overview](https://commonwealth-demo.herokuapp.com/1inch/overview)
  * examples of 4: [Profile](https://commonwealth-demo.herokuapp.com/profile/id/121490)
  * the input tag (5) can be seen in the search bar at the top of the pages above

### CODE REVIEW

**Comment by Miaplacidus** [comment]
@jessmart1213 @zakhap 
Deployment to the demo branch failed. Will try re-deployment next week!

**Comment by CowMuon** [comment]
Has this been redeployed? Can we get an update here please?

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_tag.tsx
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_tag.tsx:1]: can you rename cw_tag to CWTag, and same for the scss, please?

File: packages/commonwealth/client/styles/components/component_kit/cw_tag.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import '../../shared';
 
-.Tag {
+.Tag.Old {
```
Comment (masvelio) [inline packages/commonwealth/client/styles/components/component_kit/cw_tag.scss:3]: Let's remove `component_kit/cw_tag.scss` along with `cw_tag.tsx`. Old tags are not used anymore so we do not need them in the codebase

**Comment by Miaplacidus** [comment]
@CowMuon This has been successfully deployed to the demo branch!

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_tag.tsx
Comment (Miaplacidus) [reply packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_tag.tsx:1]: Done! Also, both have been moved into their own directories in the new_designs folder.

File: packages/commonwealth/client/styles/components/component_kit/cw_tag.scss (hunk 1 above)
Comment (Miaplacidus) [reply packages/commonwealth/client/styles/components/component_kit/cw_tag.scss:3]: Done!

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_tag.tsx
Comment (Miaplacidus) [reply packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_tag.tsx:1]: Done!

File: packages/commonwealth/client/scripts/views/components/Profile/ProfileActivityRow.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ const ProfileActivityRow = ({ activity }: ProfileActivityRowProps) => {
           </a>
         </CWText>
         <div className="dot">.</div>
-        <CWTag label={author.slice(0, 5)} />
+        <CWTag label={author.slice(0, 5)} type='disabled' />
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/Profile/ProfileActivityRow.tsx:77]: again, problem with prettier formatting. This should be double quote

**Review by masvelio — APPROVED** [review]
image

Not sure if this is related, but in components view the dropdowns that use tags are broken.

So, prettier and this ☝️ . Apart from that, looks good to me, so I am approving in advance.

**Comment by Miaplacidus** [comment]
> image
> 
> Not sure if this is related, but in components view the dropdowns that use tags are broken.
> 
> So, prettier and this ☝️ . Apart from that, looks good to me, so I am approving in advance.

@masvelio 
That brokenness is related to different work and will be dealt with. Tags are good to go both on demo and locally.

**Comment by jessmart1213** [comment]
### Design review

@Miaplacidus I have the tiniest piece of feedback but everything else looks great! 

---

Update spacing or gap for "Address tag" to be 8px instead of 10px. In the figma it's a little confusing because theres an extra parent div with 10px, so I see why you would make it 10 instead of 8px.

Screen Shot 2023-10-13 at 9 54 42 AM

Once the this update has been implemented, because its so small theres no need to inform me so that I can check it. You can move on to your next steps. 

---

Note: Also I want to say that some of the tags in the designs have font, Body 02, instead of, caption medium, so technically that didn't match designs, but I don't know much about the reasoning for that as I didn't create these tags. I prefer  for our tags to have consistency so I will update the tag designs to ensure all tags have font Caption medium.

**Review by jessmart1213 — APPROVED** [review]
Design review
@Miaplacidus I have the tiniest piece of feedback and then Design Sign-off. 

---

Update spacing or gap for "Address tag" to be 8px instead of 10px. In the figma it's a little confusing because theres an extra parent div with 10px, so I see why you would make it 10 instead of 8px.

Screen Shot 2023-10-13 at 9 54 42 AM


Once the this update has been implemented, because its so small theres no need to inform me so that I can check it. You can move on to your next steps.

---

Note: Also I want to say that some of the tags in the designs have font, Body 02, instead of, caption medium, so technically that didn't match designs, but I don't know much about the reasoning for that as I didn't create these tags. I prefer for our tags to have consistency so I will update the tag designs to ensure all tags have font Caption medium.

**Review by rbennettcw — APPROVED** [review]

**Pull request merged**